internal/controller/components/kueue/kueue_controller.go

```diff
@@ -104,6 +110,16 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 			deploy.WithCache(),
 		)).
 		WithAction(deployments.NewAction()).
+		WithAction(func(ctx context.Context, rr *types.ReconciliationRequest) error {
+			kueueCRInstance, ok := rr.Instance.(*componentApi.Kueue)
+			if !ok {
+				return fmt.Errorf("resource instance %v is not a componentApi.Kueue)", rr.Instance)
+			}
+			if kueueCRInstance.Spec.KueueManagementSpec.ManagementState == operatorv1.Unmanaged {
+				rr.Conditions.MarkFalse(status.ConditionDeploymentsAvailable, conditions.WithSeverity(common.ConditionSeverityInfo))
+			}
+			return nil
+		}).
```

Comment:
is the plan (when Unmanaged) to skip creation of any resources from Kueue's manifests 
or just mark the status on kueue? 
if it is the later,  
```
for _, deployment := range deployments.Items {
		if deployment.Status.ReadyReplicas == deployment.Status.Replicas && deployment.Status.Replicas != 0 {
			ready++
		}
	}
```
will still be run , right? so why not add it in the deployments.run() ?

Reply:
If I got what you are saying, the `deployments` action is a shared one, it does not have any clue about kueue so we can't change it to support this case. For the `unmanaged` case, we don't check the OpenShift Kueue operator deployments and we'd need to find a better way to handle this case, but we still have to set the `DeploymentsReady` condition

Reply:
ok, i see,
for the Unmanged, rr.Manifests = append(rr.Manifests, kueueConfigExtraManifestsPath()) might still be needed for 4.17+ and DeploymentReady still need to be set in DSC.status for kueue.

Reply:
the VAP is replaced by a platform webhook  and there's no more kueue related deployments in the application namespace, so here is to state that deployments are not ready because we are in the unmanaged case, but also that this status=false does not contribute to the component readiness (since severity==info)